db/migrate/20221128192313_add_participant_id_to_teams_users.rb

```diff
@@ -0,0 +1,22 @@
+class AddParticipantIdToTeamsUsers < ActiveRecord::Migration[5.1]
+  def change
+    add_column :teams_users, :participant_id, :integer, index: true
+    # add_reference :teams_users, :participants
+    add_foreign_key :teams_users, :participants
+
+    teams_users = TeamsUser.all
+    teams_users.each do |teams_user|
+      if !teams_user.team_id.nil?
+        team = Team.find(teams_user.team_id)
+        if !team.nil?
+          participant = Participant.find_by(user_id: teams_user["user_id"], parent_id: team["parent_id"])
+          if !participant.nil?
+            teams_user.participant_id = participant["id"]
+            teams_user.save
+          end
+        end
+      end
+      next unless teams_user.participant_id
```

Comment:
I'm not sure what it does

Reply:
It is a condition to check if the teams_user.participant_id is not nil before proceeding with the next user in teams_users table.